completion.go

```diff
@@ -17,29 +17,46 @@ var (
 // GPT3 Models are designed for text-based tasks. For code-specific
 // tasks, please refer to the Codex series of models.
 const (
-	GPT432K0613             = "gpt-4-32k-0613"
-	GPT432K0314             = "gpt-4-32k-0314"
-	GPT432K                 = "gpt-4-32k"
-	GPT40613                = "gpt-4-0613"
-	GPT40314                = "gpt-4-0314"
-	GPT4                    = "gpt-4"
-	GPT3Dot5Turbo0613       = "gpt-3.5-turbo-0613"
-	GPT3Dot5Turbo0301       = "gpt-3.5-turbo-0301"
-	GPT3Dot5Turbo16K        = "gpt-3.5-turbo-16k"
-	GPT3Dot5Turbo16K0613    = "gpt-3.5-turbo-16k-0613"
-	GPT3Dot5Turbo           = "gpt-3.5-turbo"
-	GPT3TextDavinci003      = "text-davinci-003"
-	GPT3TextDavinci002      = "text-davinci-002"
-	GPT3TextCurie001        = "text-curie-001"
-	GPT3TextBabbage001      = "text-babbage-001"
-	GPT3TextAda001          = "text-ada-001"
-	GPT3TextDavinci001      = "text-davinci-001"
+	GPT432K0613           = "gpt-4-32k-0613"
+	GPT432K0314           = "gpt-4-32k-0314"
+	GPT432K               = "gpt-4-32k"
+	GPT40613              = "gpt-4-0613"
+	GPT40314              = "gpt-4-0314"
+	GPT4                  = "gpt-4"
+	GPT3Dot5Turbo0613     = "gpt-3.5-turbo-0613"
+	GPT3Dot5Turbo0301     = "gpt-3.5-turbo-0301"
+	GPT3Dot5Turbo16K      = "gpt-3.5-turbo-16k"
+	GPT3Dot5Turbo16K0613  = "gpt-3.5-turbo-16k-0613"
+	GPT3Dot5Turbo         = "gpt-3.5-turbo"
+	GPT3Dot5TurboInstruct = "gpt-3.5-turbo-instruct"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
+	GPT3TextDavinci003 = "text-davinci-003"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
+	GPT3TextDavinci002 = "text-davinci-002"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
+	GPT3TextCurie001 = "text-curie-001"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
+	GPT3TextBabbage001 = "text-babbage-001"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
+	GPT3TextAda001 = "text-ada-001"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
+	GPT3TextDavinci001 = "text-davinci-001"
+	// Deprecated: Will be shut down on January 04, 2024. Use gpt-3.5-turbo-instruct instead.
 	GPT3DavinciInstructBeta = "davinci-instruct-beta"
-	GPT3Davinci             = "davinci"
-	GPT3CurieInstructBeta   = "curie-instruct-beta"
-	GPT3Curie               = "curie"
-	GPT3Ada                 = "ada"
-	GPT3Babbage             = "babbage"
+	// Deprecated: Will be shut down on January 04, 2024. Use davinci-002 instead.
+	GPT3Davinci    = "davinci"
```

Comment:
Hey, thank you for the PR. I've re-read OpenAI post and I think we might not need to warn about these non-numerated models:

> Applications using the stable model names for base GPT-3 models (ada, babbage, curie, davinci) will automatically be upgraded to the new models listed above on January 4, 2024. The new models will also be accessible in the coming weeks for early testing by specifying the following model names in API calls: ada-002, babbage-002, curie-002, davinci-002.

https://openai.com/blog/gpt-4-api-general-availability

Reply:
@sashabaranov Fix it.